omero/sysadmins/admins-with-restricted-privileges.txt

```diff
@@ -0,0 +1,418 @@
+    their :term:`Sudo` privilege, the prevention of escalation means
+    they will be able to execute maximally the privileges level they
+    were enjoying before sudoing, no more privileges can be acquired
+    by the sudoing itself. In case the sudoed-as user has less privileges
+    than the sudoing restricted admin, these lesser privileges will apply
+    also for the restricted admin as long as sudoed-in.
+
+.. _Workflow 1:
+
+Workflow 1: Data viewer
+-----------------------
+Even if you do not give any explicit privileges to the administrator with
+restricted privileges, this new administrator has some useful powers.
+These include browsing and viewing all the data of all users in all
+groups (including the groups where he/she is not a member).
+The administrator with restricted privileges
+is also able to Download all the data in all types of groups.
+Further, he/she can view user and group information, such as usernames,
+e-mail addresses, group permission levels and lists of all users
+and groups. He/she is not able to annotate, edit or delete any of the
+data or change any user or group informations though. Note that any
+administrator with restricted privileges described below or otherwise
+created combining the privileges at will would be able to perform the
+Data viewer workflow as well.
+
+Workflow 1: Data viewer in clients:
+
+- OMERO.insight: is not designed to show any groups or data you are not
+  a member of. The Data Viewer workflow is preferably
+  executed using OMERO.web or CLI
+
+- OMERO.web: viewing and downloading the data,
+  see :help:`Viewing Data <viewing-data>`
+
+- CLI: listing all images, groups and users, download::
+
+    # List all users on server
+    $ bin/omero user list
+    # List all groups on server
+    $ bin/omero group list
+    # List all images on server
+    $ bin/omero fs images
+
+.. _Workflow 2:
+
+Workflow 2: Importer
+--------------------
+The Importer's task is to import images into OMERO for other users,
+i.e. in such a manner that the imported images are owned by the users
+in OMERO, not by the Importer. This is typically an imaging facility
+manager who is importing data acquired by users on microscopes into OMERO.
+
+Importer workflow can be achieved fully with only the :term:`Sudo`
+privilege (first line in the above table). This privilege allows them
+factically to "become" the user they are importing the data for.
+There might be reasons though why the Importer needs to re-organize
+the imported data. For example, he/she made a mistake, Sudoed as
+a wrong user in a wrong group and needs to rectify the mistake using
+the command line interface (CLI) client. Whilst being sudoed, the
+Importer can Delete the wrongly imported data (even without Delete
+privilege given, see the Note above), logout, login and :term:`Sudo` as
+the correct user and repeat the import process. In short, whilst Sudoed,
+Importer can do any action which the user he/she is sudoed in as is
```

Comment:
"the user he/she is sudoed in as is" could be "the user they are becoming" or "the user they changed to".  "sudoed in as" is to me self-referential and not fully descriptive.